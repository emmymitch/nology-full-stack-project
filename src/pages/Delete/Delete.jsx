import "./Delete.scss";

const Delete = () => {
    return (
        <h1>Delete</h1>
    )
}

export default Delete;