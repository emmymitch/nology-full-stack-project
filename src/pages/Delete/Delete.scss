@use "../../styles/variables" as *;

.delete{
    display: flex;
    flex-direction: column;
    align-items: center;
    max-width: 100vw;

    & > * {
        margin: 1rem 0;
    }

    &__check{
        margin-bottom: 2rem;

        display: flex;
        flex-direction: column;
        align-items: center;
        & * {
            margin: 0.5rem 0;
        }
    }

    &__changeable{
        &--myth-card{
            margin: 0 2rem;
        }
    }
}

@media screen and (min-width: $desktop-breakpoint) {
    .delete{
        &__changeable{
            display: flex;
        }
    }
}