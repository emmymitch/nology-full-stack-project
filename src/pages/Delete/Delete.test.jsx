import Delete from "./Delete";
import { render } from "@testing-library/react";

it("Should render the Delete page", () => {
  const container = render(<Delete />)

  expect(container).toMatchSnapshot();
});