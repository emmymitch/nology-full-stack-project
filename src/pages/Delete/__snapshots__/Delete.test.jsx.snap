// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Delete page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="delete"
      >
        <h1>
          Delete
        </h1>
        <h3>
          Has something gone too horribly wrong to fix? Delete it from this page.
        </h3>
        <form>
          <div
            class="delete__check"
          >
            <label
              for="deleteSearch"
            >
              Card ID:
            </label>
            <input
              class="text-input "
              name="deleteSearch"
              placeholder="Find the unique ID at the bottom of each card"
              type="text"
              value=""
            />
            <button
              class="button "
              type="submit"
            >
              Submit
            </button>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="delete"
    >
      <h1>
        Delete
      </h1>
      <h3>
        Has something gone too horribly wrong to fix? Delete it from this page.
      </h3>
      <form>
        <div
          class="delete__check"
        >
          <label
            for="deleteSearch"
          >
            Card ID:
          </label>
          <input
            class="text-input "
            name="deleteSearch"
            placeholder="Find the unique ID at the bottom of each card"
            type="text"
            value=""
          />
          <button
            class="button "
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
